### PR TITLE
Fixes #109 Update validation and qualification (V11)

### LINKED ISSUE #109: Update validation and qualification (V11)

### PR DESCRIPTION
**Pull request by Yuri05**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ While PK-Sim® is based on a whole-body concept, the focus of its counterpart, M
...
...ology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-formulations), [**Administration Protocols**](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-administration-protocols), [**Events**](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-events), **Observers** and [**Observed Data**](https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/import-edit-observed-data). Building blocks from these groups are combined to produce a [model](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations). The advantage of building blocks is that they can be reused. For example, after having established a model for a drug after single dose intravenous administration to an animal species, just substitute the individual by a suitably parameterized virtual human population and obtain a first in man simulation model. Further substitute the formulation, to obtain a controlled-release per oral simulation model, substitute the protocol to obtain a multiple dose simulation model, or substitute the compound to obtain a simulation model for another drug.
```
Comment (Yuri05) [inline README.md:25]: just added [**Expression Profiles**](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-profile) into the list of PK-Sim BBs

**Comment by Yuri05** [comment]
@msevestre Please review when you have time

**Review by msevestre — APPROVED** [review]
Looks good to me
It should only be merged however when the setup can be downloaded. Otherwise, the notification will popup in PK-Sim....

**Comment by Yuri05** [comment]
>It should only be merged however when the setup can be downloaded. Otherwise, the notification will popup in PK-Sim....

But only when merging in master, right? Merging into develop will not change the notification

**Comment by msevestre** [comment]
oh yes sorry. Merging then

**Pull request merged**